emulator/clients/common.h

```diff
@@ -0,0 +1,29 @@
+#ifndef EMULATOR_CLIENT_COMMON_H
+#define EMULATOR_CLIENT_COMMON_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define CLIENT_PORT (8080)
+#define HIGH (1)
+#define LOW  (0)
+
+struct PinEvent {
```

Comment:
I always prefer to do
```
typedef struct {
  uint32_t pin;
  uint32_t in_n_cycles;
  bool polarity;
} pin_event_t

Then you dont have to write struct everywhere

Reply:
Yes, I guess that would be more in-line with the already established coding convention in the repo. I'll change it :)

I just personally prefer not to typedef, since it makes the types more opaque.